src/map/lua/luautils.cpp

```diff
@@ -2093,36 +2104,122 @@ namespace luautils
     {
         TPZ_DEBUG_BREAK_IF(PSpell == nullptr);
 
-        lua_prepscript(
-            PSpell->getSpellGroup() == SPELLGROUP_BLUE ? "scripts/globals/spells/bluemagic/%s.lua" :
-            PSpell->getSpellGroup() == SPELLGROUP_TRUST ? "scripts/globals/spells/trust/%s.lua" :
-            "scripts/globals/spells/%s.lua", PSpell->getName()
-        );
+        /**
```

Comment:
Why stop at a partial implementation (hack) and not force all to use the new system.  You get a broader range of spells to test out and find bugs related to them.  Assuming this system is to replace the old all spells will need to be tested at some point.

Reply:
Definitely will be. I stopped at partial because I didn't have a script to automate grabbing all the spell values and dumping them into a table.